api.go

```diff
@@ -186,9 +186,24 @@ func (s *Stats) Zero() bool {
 		s.Wait > 0)
 }
 
+// Progress contains information about the global progress of the running search query.
+// This is used by the frontend to reorder results and emit them when stable.
+type Progress struct {
+	// Priority of the shard that was searched.
+	ShardPriority float64
+
+	// Maximum priority of a shard that is being searched in parallel. This is used to reorder results
```

Comment:
minor, but normally a godoc comment starts with the field/type name. EG here it would start "MaxPendingShardPriority is ..."